test/metabase/api/user_test.clj

```diff
@@ -347,6 +347,31 @@
       :is_superuser true})
     (superuser-and-admin-pgm-info email)))
 
+(deftest create-user-mixed-case-email
+  (testing "POST /api/user/:id"
+    (testing "can create a new User with a mixed case email and the email is normalized to lower case"
+      (let [user-name (random-name)
+            email     (tu/random-email)]
+        (is (= email
+              (:email (et/with-fake-inbox
+                   (try
```

Comment:
The `try` form here should be indented 2 spaces into the `with-fake-inbox` form e.g.

```clj
(et/with-fake-inbox
  (try
    ...))
```

https://guide.clojure.style/#body-indentation is a good explanation. `with-fake-inbox` is a macro with a `body` of forms it wraps which should be indented 2 spaces